independent-publisher-connectors/icanhazdadjoke/apiDefinition.swagger.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "icanhazdadjoke",
+    "description": "With this connector you can get a random dad joke or search for a dad joke.",
+    "version": "1.0",
+    "contact": {
+      "name": "Daniel Laskewitz",
+      "url": "https://youtube.com/daniellaskewitz",
+      "email": "daniel@laskewitz.io"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://youtube.com/daniellaskewitz"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://youtube.com/daniellaskewitz"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Lifestyle and Entertainment"
+    }
+  ],
+  "host": "icanhazdadjoke.com",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "id",
+                  "title": "ID",
+                  "x-ms-visibility": "internal"
+                },
+                "joke": {
+                  "type": "string",
+                  "description": "joke",
+                  "title": "Joke"
+                },
+                "status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "status",
+                  "title": "Status",
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Fetch a random dad joke",
+        "description": "Fetch a random dad joke from icanhazdadjoke.com",
+        "operationId": "FetchRandomJoke",
+        "parameters": [
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/search": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "current_page": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "current_page",
+                  "title": "Current page"
+                },
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "limit",
+                  "title": "Limit"
+                },
+                "next_page": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "next_page",
```

Comment:
@Laskewitz Can you please update the description to `Next page`.

Can you please update all the similar description values of properties with valid ones in the swagger.